source/JsMaterialX/JsMaterialXView/webpack.config.js

```diff
@@ -9,18 +17,26 @@ module.exports = {
   },
   mode: "development",
   plugins: [
+    new HtmlWebpackPlugin({
+      templateParameters: {
+        materials
+      },
+      template: 'index.ejs'
+    }),
     new CopyPlugin({
       patterns: [
         { 
           context: "../../../resources/Images",
           from: "*.jpg", 
           to: "Images",
         },
+        { from: "./public", to: 'public' },
         { from: "../../../resources/Images/greysphere_calibration.png", to: "Images" },
-        { from: "../../../resources/Geometry/shaderball.obj",  to: "Geometry"},
+        { from: "../../../resources/Geometry/shaderball.glb",  to: "Geometry"},
+        { from: "node_modules/three/examples/js/libs/draco",  to: "draco"},
```

Comment:
I couldn't find where this is used. Do we need to copy it?

Reply:
yeah, sadly. It needs to load the wasm files. Actually we had a similar issue with Karma and we were wondering how other libraries do :/

Reply:
This is used in the index.js `dracoLoader.setDecoderPath( '/draco/' );`